.gitlab-ci.ornl.yml

```diff
@@ -0,0 +1,80 @@
+
+stages:
+  - build
+  - test
+  - clean
+
+Spack Build:
+  stage: build
+  tags:
+    - nobatch
+  needs: []
+  variables:
+#    CUSTOM_CI_BUILDS_DIR: "/gpfs/wolf/proj-shared/csc449/ci/${CI_PIPELINE_ID}"
+    WORK: "/gpfs/wolf/proj-shared/csc449/ci/"
+  script:
+    - pwd
+    - env | grep CI_
+    - export run_id=$CI_PIPELINE_ID
+    - export branch=$CI_COMMIT_BRANCH
+    - export url="https://sdk.testing.exaworks.org/result"
+    - export test="Spack"
+    - export location="ornl-$(echo $CI_RUNNER_TAGS | cut -f 1 -d ',')"
+    - export contact="andre@merzky.net"
+#    - export imnumber="LLNL-MI-834241"
+    - python3 ci/tests/test.py -s
+    - test -d $WORK/spack || git clone -c feature.manyFiles=true https://github.com/spack/spack.git $WORK/spack
+    - . $WORK/spack/share/spack/setup-env.sh
+    - spack install py-pytest
+    - python3 ci/tests/test.py -n 'spack build' -c "spack install exaworks"
```

Comment:
Do we want the Spack Build to be a test that we upload the the dashboard?

Reply:
Given that this is the SDK test suite I would argue that deployment is an important part.  But I am open to removing it also if you think that removes clutter...